RSEConUK2019-Cylc-Talk/index.html

```diff
@@ -403,165 +404,378 @@
 
     <!-- Section 2 - Cylc Usage -->
     <div
-      id="usage-of-cylc"
-      class="step"
+      id="usage-and-community"
+      class="step full vflex"
       data-rel-y="1h"
       data-rel-to="introduction-to-cylc"
       >
-      <b>Usage</b> & the Cylc <b>Community</b>
+      <p>
+        <b>Usage</b> & the Cylc <b>Community</b>
+      </p>
+            <!--
+              'community' clipart (public domain):
+              https://publicdomainvectors.org/en/free-clipart/People-sketch-in-a-circle/80562.html
+            -->
+      <embed
+        class="flexgrow"
+        src="media/decorative/community.svg"
+        style="height:100px;align:center"
+      />
     </div>
 
     <div
-      class="step"
+      id="major-sites-map"
+      class="step full vflex"
       data-rel-x="1w"
       data-rel-y="0"
-      >
-      <em>International</em> use (& development)!<br>
-       <b>Institutional</b> usage over 5 continents:
+       >
+      <p><em>International</em> use [<img
```

Comment:
That is understandable, thanks for raising it. I wasn't entirely happy with how this was written but was struggling for another way to express it. All I am trying to say is that the development sites are also user sites (as they are all major ones so I wanted to avoid any chance people might interpret them as dev-only locations). But if that is not clear I will think further about a better way to convey so.

Reply:
I've changed this now, give me a shout if it is still not clear (in the worst case I can clarify verbally in a sentence, the difficulty here is in conveying it in a short one to fit on the slides without taking up too much space as to dominate).

Reply:
Ok, how about:

International use :white_circle: & development :red_circle:.